Snippets/Core/Core_6/Headers/OverrideConversationId.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Threading.Tasks;
+using Core6.Handlers;
+using NServiceBus;
+
+public class ConversationIdHeaderOverride
+{
+    public async Task Handle()
+    {
+        IMessageHandlerContext context = null;
+        #region override-conversation-id
+        var sendOptions = new SendOptions();
+        sendOptions.SetHeader(Headers.ConversationId, "MyCustomConversationId/" + System.Guid.NewGuid());
```

Comment:
I think there is no scenario (except 3rd party integration) where an incoming message doesn't have a conversation id present. Given that this will fail, it doesn't seem to be a valid snippet to set a conversation id in a handler.

Reply:
@timbussmann You are right, this was still under the assumption that is would work. I'll update the interface.